runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -32,7 +32,9 @@
 import com.google.inject.Inject;
 import com.google.inject.Injector;
 import java.applet.Applet;
+import java.awt.*;
```

Comment:
We prefer to not have wildcard imports

Reply:
ah must've been the IDE, I'll revert that